plugins/inputs/chrony/chrony.go

```diff
@@ -47,7 +63,7 @@ func (c *Chrony) Init() error {
 			return fmt.Errorf("parsing server address failed: %w", err)
 		}
 		switch u.Scheme {
-		case "unix":
+		case "unix", "unixdram":
```

Comment:
Are we sure that this **has to be** a dgram socket? Or is both possible? If the latter, we should implement both and let the user decide...

Reply:
hmm I swear this worked as I had it set up locally when we landed #14629, however I do see an error about the type.

```
2024-06-26T19:31:30Z E! [agent] Starting input inputs.chrony: dialing "unix:///run/chrony/chronyd.sock" failed: dial unix /run/chrony/chronyd.sock: connect: protocol wrong type for socket
```

I think what I was using the UDP port via `udp://127.0.0.1:323`. I have confirmed the same behavior on Ubuntu Bionic and Ubuntu Focal as well, so let's remove the old support.

Reply:
Ok, as the past code seems to never have worked, even though I could swear I tested it and ported it from old code, I would say we just convert the `unix` to dgram and **not** introduce `unixdgram`. @powersj what do you think?

Reply:
Can you try with the `serverstats` plugin please ? Because it's the only one that needs the socket :-)

Reply:
@srebhan agreed

@Frankkkkk yep looks like that fails currently as well with `got status UNAUTH (2)`.